tests/qnodes/test_qnode_cv.py

```diff
@@ -22,7 +22,7 @@
 from pennylane.operation import CVObservable
 from pennylane.qnodes.base import QuantumFunctionError
 from pennylane.qnodes.cv import CVQNode
-
+pytestmark = pytest.mark.usefixtures("non_tape_mode_only")
```

Comment:
Do we already have tape mode tests matching these? Or is this on our todo?

Reply:
Yep, all the tests in `tests/tape/*` I ported over from the `tests/qnodes` and `tests/interfaces` folders 🙂 and improved them at the same time.

So we are safe to completely delete these two folders.